client/me/reauth-required/index.jsx

```diff
@@ -26,14 +27,17 @@ import observe from 'lib/mixins/data-observe';
 /* eslint-enable no-restricted-imports */
 import { recordGoogleEvent } from 'state/analytics/actions';
 import userUtilities from 'lib/user/utils';
+import SecurityKeyForm from 'me/reauth-required/security-key-form';
+import { getCurrentUserId } from 'state/current-user/selectors';
 
 /**
  * Style dependencies
  */
 import './style.scss';
+import TwoFactorActions from 'me/reauth-required/two-factor-actions';
 
 // autofocus is used for tracking purposes, not an a11y issue
-/* eslint-disable jsx-a11y/no-autofocus */
+/* eslint-disable jsx-a11y/no-autofocus, react/prefer-es6-class, jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid */
```

Comment:
That's fine with me, however I think the `ci/circleci: lint-and-translate` check will fail. Is that ok for merging this PR once it's accepted?

Reply:
As long as we're not introducing any of the errors/warnings that it's failing for, we are good. Ideally we should be fixing them as we go, but it would make this PR huge, so I'm fine with ignoring them here.

Reply:
The lint warnings are all related to existing code (mostly [this](https://github.com/Automattic/wp-calypso/blob/1835f94661068d5d10323fd42608596bb9064a01/client/me/reauth-required/index.jsx#L202)). I agree we should be fixing them, but IMO that should be part of another PR.

Reply:
Agreed 👍